docs/src/pages/index.js

```diff
@@ -129,9 +129,10 @@ const Home = ({ sponsors }) => {
                     <p className="mt-2  lg:mt-4 text-base xl:text-lg lg:leading-normal leading-6 text-gray-600">
                       Get notified of the incidents through your favourite tools
                       including by e-mail via SMTP, mailgun, and sendgrid,
-                      webhook, telegram, or through slack incoming webhook. More
-                      notification channels are coming including Microsoft
-                      Teams!
+                      webhook, telegram, slack incoming webhook or through your
+                      whatsapp number by registering to Monika Whatsapp
+                      Notifier. More notification channels are coming including
+                      Microsoft Teams!
```

Comment:
can delete this "More notification channels are coming including Microsoft Teams!"